vendor.conf

```diff
@@ -37,11 +37,18 @@ github.com/opencontainers/runc v1.0.0-rc2
 # OCI runtime spec, latest release as of 12/16/2016
 github.com/opencontainers/runtime-spec v1.0.0-rc3
 # logrus, latest release as of 12/16/2016
+# FIXME: github.com/docker/docker/pkg/archive requires this pkg as github.com/Sirupsen/logrus.
+# So we vendor github.com/Sirupsen as well.
```

Comment:
I agree that we must get all vendored-in content to the new name. However, just to frame that it may not be exceedingly quick to do so [although the problem here in this project may be limited to a few pkg/* files?]:

193 files in docker/docker need the capitalization change.. fairly easy PR to create, and I'm willing to do so over there.

But, the following vendors (at least at the version imported to docker/docker today) also are using capital-S Sirupsen:

docker/distribution
docker/go-connections
docker/go-events
docker/libnetwork
docker/notary
docker/swarmkit
opencontainer/runc
Microsoft/hcsshim
Azure/go-ansiterm

So, each of those also need a PR merged and then a re-vendor (or just a re-vendor in case the `master` of those projects has fixed this since the last vendor update to docker/docker).